### DISCUSSION TITLE
Can a vlan in NetBox be associated with a site and a vlan-group?

### DISCUSSION THREAD

**Discussion by steve-rathbun** [netbox-community/netbox#10345] [Q&A]
Can a vlan in NetBox be associated with a site and a vlan-group? In some older versions of Netbox I believe this was possible, but it seems something has changed.

site,group,vid,name,status
ACME-Site1,ACME-Vlans,250,ACME-TestVlan-v250,active

Row 1 group: VLAN is assigned to group ACME-Vlans (scope: None); cannot also assign to site ACME-Site1.

**Comment by joaolucasmacedo** [comment]
A VLAN can be associated with a site OR a VLAN Group.
Each VLAN Group has a scope, that may be associated to site OR other objects.
